components/EditProfilePage/ProfileSettingsModal.tsx

```diff
@@ -100,8 +98,16 @@ export default function ProfileSettingsModal({
       aria-labelledby="notifications-modal"
       centered
     >
-      <Modal.Header closeButton>
+      <Modal.Header>
```

Comment:
Bootstrap's closeButton function was throwing an illegal error when the modal was invoked from the "View Profile" page

swapped it out for something that didn't throw errors